chef.gemspec

```diff
@@ -47,7 +47,7 @@ Gem::Specification.new do |s|
   s.add_dependency "bundler", ">= 1.10"
 
   s.bindir       = "bin"
-  s.executables  = %w{ chef-client chef-solo knife chef-shell chef-apply chef-resource-inspector }
```

Comment:
Do we really need to remove chef-resource-inspector and chef-shell?

Reply:
Per discussion in standup, i decided those didn't matter (we have to do chef-client and chef-solo which is where the user pain will all come from) and so removing all the chef-* binstubs optimizes for consistency and clarity and making upper management happy while not inflicting substantially more pain than was already there.  And discussions with management have revolved around removing `chef*` (where there's a trademark issue) being the dividing line.  So I just didn't want to die on the hill of trying to argue that chef-resource-inspector needed to be kept, because I kinda think that it doesn't...